clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/vmimage/view/AzureVMImageLookupController.groovy

```diff
@@ -61,15 +62,31 @@ class AzureVMImageLookupController {
   List<AzureNamedImage> getVMImage(@PathVariable String account, @PathVariable String region, @PathVariable String imageId) {
```

Comment:
Ah I see. So where will the user specify the image name to be deployed in the pipeline workflow? Usually it's done with a bake, or a findImage stage. findImage will resolve a server group and its image, which should provide enough details to specify what will be deployed.